uiux/challenge-1/src/assets/styles/main.scss

```diff
@@ -0,0 +1,102 @@
+$flexboxgrid-grid-columns: 12;
+$flexboxgrid-gutter-width: 1rem;
+$flexboxgrid-outer-margin: 1rem;
+
+$fa-font-path: '/fonts';
```

Comment:
The path is correct. It can works if we refer the fonts in `node_modules` directly, compile, change back the path, recompile, and it works, so probably something to do with my `webpack.config.js`.